Capstone/View Controllers/DetailsViewController.h

```diff
@@ -0,0 +1,21 @@
+//
+//  DetailsViewController.h
+//  Capstone
+//
+//  Created by Abel Kelbessa on 7/5/22.
+//
+
+#import "ViewController.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface DetailsViewController : ViewController
+
+@property (strong, nonatomic) NSString *detailName;
+@property (strong, nonatomic) NSString *detailImage;
+@property (strong, nonatomic) NSString *detailLocation;
+@property (strong, nonatomic) NSString *detailDetail;
```

Comment:
Feels like this might be served with a data model object (e.g. a College.h model class)